### ISSUE TITLE
Allow changing the financing option in the checkout view

### ISSUE THREAD

**Issue by tommygonzaleza** [breatheco-de/breatheco-de#7852]
We should add a dropdown with different financing options of this plan.

This is the figma: https://www.figma.com/design/hJsE2D6fu3peR49mt6kZD1/4Geeks.com?node-id=14751-53746&t=sI55kaB46faESjot-1

**Comment by JennGuz** [comment]
https://github.com/breatheco-de/app/pull/1652

**Comment by KonnorThatcher** [comment]
Choosing a payment plan in the dropdown menu during the checkout process doesn't actually change the payment plan selected. It still uses the plan you entered the checkout process with.

In the video below, I went into the checkout view having selected the $110 x 5-month payment plan, then in the middle of the checkout process used the dropdown menu to change to the $500 x 1-month plan. But when I finished the checkout process and went to view my subscription, it still showed the plan that I had initially selected. Here's the [plan financing](https://breathecode-test.herokuapp.com/admin/payments/planfinancing/455/change/) of that user, which shows there's an invoice for $110 instead of $500.

https://www.loom.com/share/a2867148746e49018904d0bc14452bec?sid=ddc7b104-a346-480a-babb-1c83ecf7e6b7
